### DISCUSSION TITLE
Adjust scroll area to change volume to leftmost 10% of screen, rather than left 50%

### DISCUSSION THREAD

**Discussion by Zeno-ZX** [ppy/osu#21375] [Ideas] [answered]
Currently, when scrolling via scroll wheel with your cursor around the middle, it is very easy to accidentally quickly change volume instead of scrolling through maps as expected. I think an easy way to make this happen less, while still keeping volume control very simple is to simply adjust the usable area to adjust volume. Leftmost 10-20% of screen would make it quite intentional to change volume, rather than accidentally changing it by having your cursor barely on the left 50% and scrolling.

Edit for clarity: This applies to osu! Lazer.

**Comment by peppy** [comment, marked as answer]
I'd sooner just disable scroll volume at song select. You can already adjust by scrolling over the volume display in the toolbar.

**Reply by peppy** [reply]
https://github.com/ppy/osu/pull/21377

**Comment by LittleEndu** [comment]
Also #2238 would address this
